### ISSUE TITLE
Organize Wiki pages sidebar

### ISSUE THREAD

**Issue by sirinoks** [Seneca-CDOT/telescope#2689]
In the current Wiki we have a bunch of pages that are just there without any specific order. Look at what I mean:
<img src="https://cdn.discordapp.com/attachments/353432193730871296/933728678716772352/Screenshot_388.png" />

As you can see we have different categories mixed up. What bothers me is that many pages that are no longer relevant are mixed in with the important ones. 

How I would like it to be:

- Main page of the course
- Guides
- Most recent triage
- Older triages
- Something else
...
- No longer relevant triages/meetings/information

I would actually work on this, but I don't even know if I'm able to edit pages. I don't see where, I can only edit the pages themselves, not the sidebar.

**Comment by humphd** [comment]
I think the sidebar is auto-generated.  We'd have to investigate.  Maybe a better approach is to streamline our Home wiki page so it's easier to find things?

**Comment by sirinoks** [comment]
That works. In that case it's something I can do.

**Comment by cindyledev** [comment]
I got really excited when I saw the tiny arrow beside each of the document titles thinking they were drop-down menus for sub-categories but no, I was wrong. I thought we were finally gonna have a cool Table of Contents in the sidebar of our Telescope Wiki. C'mon GitHub, we need some way to organize our notes! Give us a way to make a Table of Contents or sub-categories or archive things or even sort!! I'll even settle for sorting!! So many options but GitHub gives us "hey, did someone say they wanted to see the headings in the markdown? we can do that ohoho"
